app/views/readings/show.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Readings#show</h1>
```

Comment:
Add the view when you are ready to implement it, each Pull Request should be a **complete** and **isolated** set of changes that go to production, not a checkpoint of your progress building the final solution